contracts/SpokePool.sol

```diff
@@ -1429,6 +1442,26 @@ abstract contract SpokePool is
         }
     }
 
+    // Re-implementation of OZ _callOptionalReturnBool to use private logic. Function executes a transfer and returns a
+    // bool indicating if the external call was successful, rather than reverting. Original method:
+    // https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/utils/SafeERC20.sol#L188
+    function noRevertTransfer(
+        address token,
+        address to,
+        uint256 amount
+    ) internal returns (bool) {
+        bool success;
+        uint256 returnSize;
+        uint256 returnValue;
+        bytes memory data = abi.encodeCall(IERC20Upgradeable.transfer, (to, amount));
+        assembly {
```

Comment:
should this be annotated as memory-safe, like in OZ?

Reply:
I think the solidity version we are using does not support this. it gave me an error when trying to do that.